pydis_site/templates/events/pages/code-jams/9/frameworks.html

```diff
@@ -71,70 +71,70 @@ <h3 id="approved-frameworks"><a href="#approved-frameworks">Approved Frameworks<
     <div class="card mb-4">
         <div class="card-content">
             <div class="content">
-                <p class="subtitle">Django Channels</p>
-                <p>Django Channels adds WebSocket-support to Django - built on ASGI like other web frameworks.
+                <p class="subtitle">AioHTTP</p>
+                <p>AioHTTP provides both a client and server WebSocket implementation, while avoiding callback-hell.
```

Comment:
The capitalization here looks weird, IO is an abbreviation that should also be capitalized. But looking through the docs, only the title of the main page had AIOHTTP capitalized while all other uses are lowercase
```suggestion
                <p class="subtitle">aiohttp</p>
                <p>aiohttp provides both a client and server WebSocket implementation, while avoiding callback-hell.
```